frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -244,8 +244,19 @@ export class MarkerPageComponent implements OnInit {
         }
     }
 
+    public clearSelector() {
+        this.marker.clearCurrentSelector();
+        this.selectorActions = [];
+    }
+
     public setTag(tag: LabelTag) {
         this.marker.setCurrentTag(tag);
+        const currentSelector = this.marker.getCurrentSelector();
+        if (!isUndefined(currentSelector)) {
+            if (!this.isToolSupportedByCurrentTag(currentSelector.getSelectorName())) {
```

Comment:
I am wondering now if we should eventually  commit to switching `Selector` to `Tool` to be consistent with backend? Now we have situation when we are calling `isToolSupported...` with `selector` as an argument.

For me it is a little bit confusing. I feel like it will only get worse if we will not do anything about that. So either we can change `selector` to `tool` or at least change methods name to `isSelectorSupportedBy...`.

Reply:
Better change this earlier than later. Maybe new issue "Make frontend consistent with backend".

Reply:
Could you create the issue, please?

I think it would be helpful to add some task there: e.g change Selector to Tool. We should also all go trough our frontend code to see if we have any other inconsistencies and add them to this issue as a task.

Reply:
@kolszewska 👍 I think that it will be good idea to make common names for such components. So how about strategy to change it? We can leave it as it is here and then globally refactor names by resolving [this issue](https://github.com/jpowie01/MedTagger/issues/389).

Reply:
I would leave it as it is here and fix it as part of the issue 💯